.github/workflows/checks.yml

```diff
@@ -21,11 +21,11 @@ jobs:
         uses: actions/cache@v3
         with:
           path: ${{ env.pythonLocation }}
-          key: ${{ runner.os }}-formatting-black-22.3.0
+          key: ${{ runner.os }}-formatting-black-23.7.0
```

Comment:
We should probably have the black version be an environment variable so we can change it in just one location. Nitpick, though, so I'm not bent on having that change in this PR.